tests/feature/feature_console/deregister.pm

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2017 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Test the function of SUSEConnect to deregister a module
+# Maintainer: Yi Xu <yxu@suse.com>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+
+sub run() {
+    select_console 'root-console';
+
+    my $count = script_output("SUSEConnect --status-text | grep -c \"^\\s*Registered\"", 200);
+    if ($count eq 0) {
+        die "There is no module registered";
+    }
+
+    # check if a certain module is registered, e.g. toolchain
+    my $module  = "sle-module-toolchain";
+    my $abbrv   = "tcm";
+    my $version = get_required_var('VERSION') =~ s/([0-9]+).*/\1/r;
```

Comment:
travis checks complains that `\1` should be `$1` instead. Please change that, then LGTM